vendor/knative.dev/eventing/test/lib/client.go

```diff
@@ -133,7 +133,7 @@ func (c *Client) runCleanup() (err error) {
 }
 
 func getTracingConfig(c kubernetes.Interface) (corev1.EnvVar, error) {
-	cm, err := c.CoreV1().ConfigMaps(resources.SystemNamespace).Get(context.Background(), configtracing.ConfigName, metav1.GetOptions{})
+	cm, err := c.CoreV1().ConfigMaps(system.Namespace()).Get(context.Background(), configtracing.ConfigName, metav1.GetOptions{})
```

Comment:
This seems the issue

Reply:
here it is we need to fix this one https://github.com/knative/eventing/commit/7eb6009b0c99ac75e2eac0beac31b72ace34ccf1#diff-fbb940d8ac3d40d6c801e8a40575b2a3cd6d6e3f1d2d8f8122fbc2c438a6dccfR67